.github/pull_request_template.md

```diff
@@ -1,24 +1,32 @@
-# Description
+## Description
 
-- Provide a summary of the modification as well as the issue that has been resolved. List any dependencies that this modification necessitates.
+Provide a summary of the modification as well as the issue that has been resolved.
 
 - Fixes # (issue)
 
 ## Changes
 
+<details>
+<summary>Describe the changes you made</summary>
+
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Refactor (non-breaking change which refactors the code base)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] This change requires a documentation update
+- [ ] Documentation update
+
+</details>
 
 ## Checklist
 
+<details>
+<summary>Ensure that you followed the following</summary>
+
+- [ ] I have added a summary of my changes to the [CHANGELOG](https://github.com/openvinotoolkit/anomalib/blob/main/CHANGELOG.md) (not for minor changes, docs and tests).
 - [ ] My code follows the [pre-commit style and check guidelines](https://github.com/openvinotoolkit/anomalib/blob/main/CONTRIBUTING.md) of this project.
```

Comment:
Minor comment but, should we remove this? Since the CI controls acceptance of PR based on code quality, this might be superfluous.

Reply:
Yeah I was thinking about that too